source/EscortDisplay.cpp

```diff
@@ -64,6 +56,10 @@ void EscortDisplay::Add(const Ship &ship, bool isHere, bool fleetIsJumping, bool
 // Draw as many escort icons as will fit in the given bounding box.
 void EscortDisplay::Draw(const Rectangle &bounds) const
 {
+	const Interface &element = *GameData::Interfaces().Get("escort element");
+
+	const int WIDTH = element.GetValue("width");
```

Comment:
Upper case constants are usually only for top-level constants with a fixed value, unlike here.